.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,303 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <option name="LINE_SEPARATOR" value="&#10;" />
+    <GroovyCodeStyleSettings>
+      <option name="STATIC_FIELDS_ORDER_WEIGHT" value="3" />
+      <option name="FIELDS_ORDER_WEIGHT" value="5" />
+      <option name="CONSTRUCTORS_ORDER_WEIGHT" value="6" />
+      <option name="METHODS_ORDER_WEIGHT" value="7" />
+      <option name="STATIC_INNER_CLASSES_ORDER_WEIGHT" value="1" />
+      <option name="INNER_CLASSES_ORDER_WEIGHT" value="2" />
+    </GroovyCodeStyleSettings>
+    <JavaCodeStyleSettings>
+      <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99999" />
+      <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99999" />
+      <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">
+        <value />
+      </option>
+      <option name="IMPORT_LAYOUT_TABLE">
+        <value>
+          <package name="" withSubpackages="true" static="true" />
+          <emptyLine />
+          <package name="java" withSubpackages="true" static="false" />
+          <emptyLine />
+          <package name="javax" withSubpackages="true" static="false" />
+          <emptyLine />
+          <package name="org" withSubpackages="true" static="false" />
+          <emptyLine />
+          <package name="" withSubpackages="true" static="false" />
+        </value>
+      </option>
+    </JavaCodeStyleSettings>
+    <ScalaCodeStyleSettings>
+      <option name="alwaysUsedImports">
+        <array>
+          <option value="io.udash._" />
+        </array>
+      </option>
+      <option name="PLACE_SELF_TYPE_ON_NEW_LINE" value="false" />
+      <option name="BLANK_LINES_AROUND_METHOD_IN_INNER_SCOPES" value="0" />
```

Comment:
I think `<option name="FORMATTER" value="1" />` should be added due to scalafmt usage.